rust/src/smb/smb.rs

```diff
@@ -2327,7 +2327,6 @@ pub unsafe extern "C" fn rs_smb_register_parser() {
         get_state_data: rs_smb_get_state_data,
         apply_tx_config: None,
         flags: APP_LAYER_PARSER_OPT_ACCEPT_GAPS,
-        truncate: Some(rs_smb_state_truncate),
```

Comment:
`rs_smb_state_truncate` seems otherwise uncalled?

Reply:
Are you asking if it's uncalled in `master`?

Reply:
I see that it is, so why keep it?